Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1378,7 +1378,7 @@ extension BrowserViewController: BrowserToolbarDelegate {
                 if isBookmarked {
                     self.removeBookmark(url)
                 } else {
-                    self.addBookmark(url, title: tab.title)
+                    self.addBookmark(url, title: self.title)
                 }
```

Comment:
Nit: Comment needed? As the bookmark is about the tab and the title, unsure why you made this change.